pcs_and_grm.R

```diff
@@ -1,78 +0,0 @@
-#! /usr/bin/env Rscript
```

Comment:
I was originally just going to have this print a warning that it was defunct, but there's enough wrong with this script that I think it makes sense to remove it